decidim-core/app/jobs/decidim/machine_translation_save_job.rb

```diff
@@ -28,6 +28,31 @@ def perform(resource, field_name, target_locale, translated_text)
         resource.update_column field_name.to_sym, resource[field_name]
         # rubocop:enable Rails/SkipsModelValidations
       end
+
+      send_translated_report_notifications(resource) if reported_resource_in_organization_language?(resource, target_locale)
+    end
+
+    private
+
+    def send_translated_report_notifications(reportable)
+      reportable.moderation.reports.each do |report|
+        Decidim::ReportedMailer.send_report_notification_to_users(reportable.moderation.participatory_space.moderators, report)
+      end
+    end
+
+    def reported_resource_in_organization_language?(resource, target_locale)
+      resource_reported?(resource) && target_locale == resource.organization.default_locale && resource_completely_translated?(resource, target_locale)
```

Comment:
I think it's better if we defend ourselves a little bit:

```suggestion
      return unless resource.try(:organization)
      
      resource_reported?(resource) && target_locale == resource.organization.default_locale && resource_completely_translated?(resource, target_locale)
```